std/xml.d

```diff
@@ -2043,9 +2043,9 @@ class ElementParser
                     const startTag = startTags[tag_.name];
                     string text;
 
-                    immutable(char)* p = startTag.tagString.ptr
+                    immutable(char)* p = &startTag.tagString[0]
                         + startTag.tagString.length;
```

Comment:
This whole thing has to be rethought. You can't do pointer arithmetic here and be `@safe`. Just changing the one aspect gets us no closer.

What you might need is `&tag_.tagString[0] - &startTag.tagString[0] + startTag.tagString.length` instead of `q-p`.